### PR TITLE
Whitepaper Chapter1 migrate from Google Docs

### PR DESCRIPTION
**Pull request by tomkivlin**
Copied from https://docs.google.com/document/d/1LO6G4sWHJKKW77z7gj2Aqt3IF3dRJL5q6Q1wXI4Kj8g/edit#

Note, this does include suggestions that hadn't necessarily been accepted (just because that's the way Google Docs copies/pastes).

### CODE REVIEW

**Comment by tomkivlin** [comment]
My suggestion would be to merge this as-is (if you're happy) and then get people to raise issues on particular sections/wording for discussion, and then once agreement/consensus is reached a PR is prepared for each issue.

**Comment by taylor** [comment]
I've contributed updates via https://github.com/tomkivlin/telecom-user-group/pull/1

File: whitepaper/chapter01.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  * [1.5.2 Software Defined Networking And The Emergence of VNFs](#1.5.2)
+  * [1.5.3 Principles for Cloud Native Telco Infrastructure and Applications](#1.5.3)
+  * [1.5.4 Evolving the Stack From VNFs to CNFs](#1.5.4)
+* [1.6 Cloud Native for Telcos in Practice](#1.6)
+* [1.7 Conclusion](#1.7)
+
+<a name="1.1"></a>
+## 1.1 Introduction
+
+Kubernetes or containers for workloads has celebrated its 5th Birthday recently , during this time all industry has seen wider support and adoption of this technology to build the infrastructure . It is logical that operators worldwide who have already embarked on an ambitious journey to transformation show interest in to it as a catalyst to support transformation . It will not only makes a perfect business senses to do so but it will enable networks and applications to be treated alike IT or web applications , a promise if fulfilled will support rapid digitisation and cloudification of networks. This is why understanding cloud native journey from an operator’s community perspective is so important.
```
Comment (taylor) [inline whitepaper/chapter01.md]: The wording needs updating in this section.  Referencing K8s and it's birthday are probably not needed.  Focus on cloud native introduction to operators.

File: whitepaper/chapter01.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+<a name="1.3.1"></a>
+### 1.3.1 What "loosely coupled" means in cloud native systems
+
+Cloud native systems work on the principle of breaking a larger monolithic application in to a fine tuned components  that is based on have a clear separation between their microservices. The separation is based on a technology like containers which will allow to deploy each of this split software component on an independent set of container infrastructure. Cloud native applications and systems generally benefit from segregating as many services as possible into different containers. Additional benefits of cloud native architectures include scalability (vertical and horizontal) per microservice. This simplifies deployment in a wide variety of infrastructure environments, ranging from endpoint device to edge station to large data centres. It also allows service providers to deploy resources in the most efficient manner possible while maintaining necessary service levels for regulated and unregulated service types.
```
Comment (taylor) [inline whitepaper/chapter01.md]: This is about building any cloud native application.  It may or may not be monolithic.  The components may not be fine tuned.  A cloud native application should use miroservices and the microservices should have clear separation.

File: whitepaper/chapter01.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+The delivery of microservice software harnesses the existing group boundaries within an organisation and works with, not against, the different rates of change and business capabilities residing within those boundaries. This allows events such as security patches to be rolled out faster and easier because they don't have to be deployed in lock-step with other parts of the application.
```
Comment (taylor) [inline whitepaper/chapter01.md]: This entire next set of paragraphs has duplication and overlap.  It needs to be reworked  and collapsed

File: whitepaper/chapter01.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+<a name="1.3.2"></a>
+### 1.3.2 Cloud native applications require orchestration
+
+Because they are loosely coupled, cloud native services are always managed by an application orchestrator, such as Kubernetes. As cloud native applications are built from a loosely coupled collection of services which are delivered via an API, this coordination function is paramount, not only for reliability but also for performance and resiliency. In addition, from an operational standpoint, an orchestrator is necessary to manage and deploy these services; manual deployment and LCM meshing of these services and applications would be complex, error-prone, and overly resource intensive.
```
Comment (taylor) [inline whitepaper/chapter01.md]: LCM meshing?

File: whitepaper/chapter01.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+<a name="1.3.3"></a>
+### 1.3.3 Infrastructure, Deployment and Configuration of Cloud Native Systems
+
+[Immutable infrastructure](https://docs.google.com/document/d/1fwV3vZB5IgHb6uUgQpHts9mFR0HheYbIrlg9hGfWYfA) (the orchestrator and all of the software and hardware that it depends on) is provisioned using baked and versioned templates (e.g. server images) or a combination of templates and bootstrapping (some repeatable and versioned process that is applied to the template e.g. Kubeadm).  The underlying infrastructure is not changed after it is made ready for use. New changes to the infrastructure are rolled out as new instances of infrastructure. By separating the application layer from the infrastructure layer, it is possible to iterate faster with minimal dependency risk. Similarly it enables the vision of programmable and software defined infrastructure that can made possible to automate end to end deployment .
```
Comment (taylor) [inline whitepaper/chapter01.md]: The last sentence has a good idea... needs rewording.

File: whitepaper/chapter01.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+In cloud native systems, the atomic unit of deployment is the container, on an immutable and decoupled compute environment that nearly completely abstracts function from the underlying physical infrastructure and network architecture. At present, the most commonly used container environment is an OCI-compliant implementation. A container is different than a Virtual Machine (VM), which was the initial base unit of cloud computing. A VM hosts a Guest OS capable of delivering a service's required compute and networking. This guest runs on top of a hypervisor consuming the virtual infrastructure and orchestration it provides. In contrast, containers run on top of a shared Host OS (with a shared kernel) kernel. This distinction is key because containerisation enables immutability, disposability and resource efficiency in ways that are not easily achievable with VMs.
+
+For Kubernetes specifically, the atomic unit of management is a Pod. A Pod is a set of containers co-located in the same or different host and sharing the same Linux networking namespace.  For telco (and other) cloud native apps, the Pod is the basic building block of deployment, scaling operations, and upgrades.
```
Comment (taylor) [inline whitepaper/chapter01.md]: This should be updated to be aligned with K8s docs https://kubernetes.io/docs/concepts/workloads/pods/pod/

File: whitepaper/chapter01.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+Cloud native applications consist of sets of cloud native microservices. The configuration of these microservices is not changed after deployment (note, this does allow for changes in application data such as "add a new user"). New features for a cloud native application and the underlying Pods are rolled out as new artefacts and new configuration. Cloud native systems are configured declaratively. This means that the system configuration declares “what” a loosely coupled system should look like, not ”how” the system should be created. The “how” of the application is determined by the tooling (e.g. the orchestrator, operators, and CRDs).
+
+<a name="1.4"></a>
+## 1.4 Cloud Native Network Functions
```
Comment (taylor) [inline whitepaper/chapter01.md]: A CNF is a cloud native concept and should focus on cloud native principles as the basis.  Supporting additional requirements in an application, system or platform is a super set after first meeting the cloud native principles.   This section should focus on that and let other documents extend this base CNCF definition

File: whitepaper/chapter01.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+
+Three types of definitions:
```
Comment (taylor) [inline whitepaper/chapter01.md]: This should be removed

File: whitepaper/chapter01.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+
+Work is ongoing to bring the most demanding and mission critical telecommunications systems to cloud native, including those that must fulfil regulatory requirements (including emergency call handling). However, in theory, CNFs running on Kubernetes should provide equivalent regulatory compliance for the same reasons that cloud native applications for enterprises tend to be more fault-tolerant, more responsive, and more resilient; because loosely coupled, immutable systems are inherently better at handling highly complex applications with strict SLAs and multiple dependent and interactive functions. This is the significant promise of cloud native for telecommunications applications and functions.
+
+Contributors: Daniel Bernier, @tomkivlin, Gergely Castari, Tamas Zsiros, Jeff Saelens, Khawaja Daniyal, Bill Mulligan, Taylor Carpenter, Wavell Watson, Frederick Kautz, Herbert Damker, Ildiko Vancsa, Dave Cremins, Henrik Saveedra Persson, Dan Kohn, Saad Sheikh, Lei Wang, Ervins Kampans
```
Comment (taylor) [inline whitepaper/chapter01.md]: Alphabetical?
Links to github?

File: whitepaper/chapter01.md (hunk 7 above)
Comment (TamasZsiros) [reply whitepaper/chapter01.md]: One thing to maybe keep in mind is that while the microservices configuration is not changed throughout their lifecycle, a telecom application (built up of many microservices) might need to expose configuration on  a higher abstraction level. For example you might want to change the general behavior of a CNF in the network (represented via a configuration management object exposed vie a nort-bound interface to a management system). This setting does not directly translate to "microservice configuration", still needs to be accessed, stored, managed, and still determimes the behavior of the CNF.

Comment (taylor) [reply whitepaper/chapter01.md]: Yes, I can see higher-level abstractions of configuration and management as something that could be desired in larger applications.  That is possible while following cloud native principles (including composability in this case).

If I'm understanding correctly, then it seems like the behaviour can be managed by the external management system directly with the CNFs cloud native API or through a "configuration management" service which communicates with the CNFs and provides a higher level of abstraction for the management system.  That configuration management service should itself be cloud native of course for the entire "larger application" to be cloud native.

The CNF modifiable run-time behaviour should be adjusted via APIs.  State for restarts, scaling, etc, should be stored outside of the CNF (eg. adjust max sessions to 10k).

**Comment by tomkivlin** [comment]
@taylor I think the merge on my fork has resolved your conversations?

File: whitepaper/chapter01.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+
+This was the same pattern followed during server virtualisation, when initial efforts focused on replicating physical servers as virtual entities. Cloud native however, requires a more fundamental redesign and rethink of how telco and backbone/core network systems function. In that sense, as well, cloud native for telco is actually not dissimilar from forklifting over monolithic legacy applications into virtual environments and then decoupling the tightly bundled functions and processes to create a more resilient, manageable, and performant infrastructure that derives immutability precisely from its ephemerality. In reality, CNFs will be introduced alongside VNFs and PNFs (physical network functions) and will need to fit into operational models that govern those earlier types of network functions. Additionally, CNFs will require the capability to be orchestrated by management systems already present in the service providers' networks.
```
Comment (jeffsaelens) [inline whitepaper/chapter01.md]: "Cloud native however, requires a more fundamental redesign and rethink of how telco and backbone/core network systems function."

> We should call out the reason for this. cloud native approaches changed how developers approach software design. As Network Functions become more software centric, they require the adoption of these paradigm shifts if they are to be cloud native themselves.

File: whitepaper/chapter01.md [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+  * [1.5.2 Software Defined Networking And The Emergence of VNFs](#1.5.2)
+  * [1.5.3 Principles for Cloud Native Telco Infrastructure and Applications](#1.5.3)
+  * [1.5.4 Evolving the Stack From VNFs to CNFs](#1.5.4)
+* [1.6 Cloud Native for Telcos in Practice](#1.6)
+* [1.7 Conclusion](#1.7)
+
+<a name="1.1"></a>
+## 1.1 Introduction
+
+There has been a steady increase in the adoption of cloud native technologies to build infrastructure and run workloads across the board, to the point that it's now the defacto choice for enterprise and IT. It makes perfect sense that the Telecom industry wants to benefit in a similar way with it's platforms, networks and telephony applications.
```
Comment (dabernie) [inline whitepaper/chapter01.md]: I would rephrase to "It then makes perfect sense for the Telecom industry to follow in order to benefit in a similar way with it's platforms ..."

[the remark this answers is not in the document]
Comment (dabernie) [reply whitepaper/chapter01.md]: +1

File: whitepaper/chapter01.md [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+
+Cloud native applications also have all of their runtime dependencies packaged with them during the build phase which is then used for deployment. This allows for individual business units and operations teams to care and maintain for their software with less risk of other stakeholders disrupting their service’s “uptime”. In practice, this allows for scenarios such as owner of service “X” being able to theoretically scale their chosen software independent of service “Y”. These potentially clear lines of demarcation provide a paradigm that makes sense to legacy operations groups who are being forced to rapidly assess how they’ve delegated roles and responsibilities.
+
```
[the remark this answers is not in the document]
Comment (dabernie) [reply whitepaper/chapter01.md]: Also inclined to reference capabilities like blue/green or canary updates for parallel versioning & rollbacks

File: whitepaper/chapter01.md [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+The orchestration layer is tasked with composing, decomposing, and reshaping or resizing the resources assigned to specific applications provided by the infrastructure. It does this based on assessments and monitoring of performance and availability of the actual infrastructure used by application. In that manner, the applications are the “brains”, directing the orchestration layer towards functional goals and outcomes (for example, low latency or specific uptime and resiliency metrics). Thus, the objective of orchestration is to create an entity that aggregates the microservices into services which can be easily mapped to network functions.
```
[the remark this answers is not in the document]
Comment (dabernie) [reply whitepaper/chapter01.md]: We also never reference the declarative nature of cloud native applications.

File: whitepaper/chapter01.md [hunk 14]
```diff
@@ -0,0 +1,140 @@
...
+
+For Kubernetes specifically, the atomic unit of management is a Pod. A Pod is a set of application containers which are 
+co-located and co-scheduled on the same machine (Node in Kubernetes terms) which run with a shared context and the same Linux networking namespace. For telco (and other) cloud native applications, the Pod is the basic building block of deployment, scaling operations, and upgrades.
+
+Cloud native applications consist of sets of cloud native microservices. The configuration of these microservices is not changed after deployment (note, this does allow for changes in application data such as "add a new user"). New features for a cloud native application and the underlying Pods are rolled out as new artifacts and new configuration. Cloud native systems are configured declaratively. This means that the system configuration describes “what” a loosely coupled system should look like, not ”how” the system should be created. The “how” of the application is determined by the tooling (e.g. the orchestrator, operators, and CRDs).
```
Comment (michaelspedersen) [inline whitepaper/chapter01.md]: Following the previous sections of the document, I believe the _"(note, this does allow for changes in application data such as "add a new user")"_ part either needs to be explained in more detail, or left out until being discussed or elaborated on further into the document. This relates to section 1.4 and the review comments added there as well (day 0 and day n configuration)

File: whitepaper/chapter01.md [hunk 15]
```diff
@@ -0,0 +1,140 @@
...
+
+In cloud native systems, the atomic unit of deployment is the container, on an immutable and decoupled compute environment that nearly completely abstracts function from the underlying physical infrastructure and network architecture. At present, the most commonly used container environment is an OCI-compliant implementation. A container is different than a Virtual Machine (VM), which was the initial base unit of cloud computing. A VM hosts a Guest OS capable of delivering a service's required compute and networking. This guest runs on top of a hypervisor consuming the virtual infrastructure and orchestration it provides. In contrast, containers run on top of a shared Host OS (with a shared kernel) kernel. This distinction is key because containerisation enables immutability, disposability and resource efficiency in ways that are not easily achievable with VMs.
```
[the remark this answers is not in the document]
Comment (dabernie) [reply whitepaper/chapter01.md]: agreed, we are already seeing instances of unikernels as a form of packaging supported on k8s with sometimes better performance then even containers compared to VMs. (ref: https://tools.ietf.org/html/draft-natarajan-nfvrg-containers-for-nfv-03)

File: whitepaper/chapter01.md (hunk 7 above)
Comment (dabernie) [reply whitepaper/chapter01.md]: In my view, and I tend to agree with the "immutable infra principles", a CNF deployment state (ie interfaces, mem, libraries, device drivers, version, etc.) cannot be modified at runtime. How it behaves when a packet enters into it could be totally reprogrammable. For example, I could run a CNF Data Plane with P4Runtime which makes its forwarding pipeline totally modifiable but which does not change its nature to the underlying infra.

File: whitepaper/chapter01.md [hunk 16]
```diff
@@ -0,0 +1,140 @@
...
+
+<a name="1.4"></a>
+## 1.4 Cloud Native Network Functions
+
+A cloud native network function (CNF) is a cloud native application that implements or facilitates network functionality. A cloud native network function consists of one or more microservices, and has been developed using Cloud Native Principles including immutable infrastructure, declarative APIs and a “repeatable deployment process”.
+
+An example of a simple CNF is a packet filter which implements a single piece of network functionality as a microservice. A firewall is an example of a CNF which may be composed of more than one microservice.
```
[the remark this answers is not in the document]
Comment (dabernie) [reply whitepaper/chapter01.md]: I have lost count on how many times we restate that a CNF is cloud native application that ... before submitting, we should revalidate how much we re-express the definition.
As for an exemple of a good CNF model, I always liked the UTM example. It would be made of a set of microservices per each behaviour (Packet-IO, FW, DPI, NAT, MGMT-API, StateDB) linked together either in "pipeline" or "snake-mode" but still compose a full UTM service

File: whitepaper/chapter01.md (hunk 10 above)
Comment (dabernie) [reply whitepaper/chapter01.md]: "CNFs will require the capability to be orchestrated by management systems already present in the service providers' networks"

CNFs and cloud-native also provide the opportunity to decentralize complexity of LCM and service resiliency from the more "top-down" orchestration systems (and I include present ONAP in this). Just look at the difference in API frameworks from OpenConfig with gRPC versus the slew of interfaces in 3GPP and ETSI-NFV.

Comment (dabernie) [reply whitepaper/chapter01.md]: What is still really important to understand that from a service perspective (ie an 5GSA core) Service Assurance still needs to be able to reference the CNF as an entity

File: whitepaper/chapter01.md [hunk 17]
```diff
@@ -0,0 +1,140 @@
...
+### 1.5.3 Principles for Cloud Native Telco Infrastructure and Applications
+
+At its core, the evolution from box-centric to process and function-centric requires an evaluation of the OSI Layers through a cloud native lens. The best general practices of cloud native infrastructure deployment and application design should also be applied to networking infrastructure Two key shifts required for CNFs to function well are deployment pipelines and the use of declarative programming and scripts in configuration. By pipelines, we mean the use of continuous integration (CI) and continuous delivery / deployment (CD) platforms. This is fundamental because it allows for rapid iteration, proper versioning, easier rollbacks, canary testing, and many other capabilities that are presently complex and often not possible in deploying VNFs. By declarative language, we mean a shift to intelligent orchestration that is better able to  maintain required service levels and maximise efficiencies based on available resources. This will ultimately mean a transition of network intelligence and design  - function, description, and configuration -  into other parts of Kubernetes (Helm Charts and CRDs).
```
Comment (dabernie) [inline whitepaper/chapter01.md]: This answers my previous comment on declarative nature

File: whitepaper/chapter01.md [hunk 18]
```diff
@@ -0,0 +1,140 @@
...
+
+Back-office applications, such as BSS and OSS functions, can run directly on Kubernetes and will enjoy all the same benefits that enterprise applications derive from Kubernetes. Operators should also be able to run real-time, critical functions on Kubernetes. CNFs must be able to  run  on Kubernetes and benefit directly and strongly from cloud native principles. It is important to note that, in theory, Kubernetes and containerisation should allow for true cloud-blind immutability and portability. This will only be true if conforming Kubernetes versions and configurations are used and supported by operators. That said, creating a conformance model for Kubernetes for telecommunications is non-trivial, requiring validation and certification of key elements such as network software (CNI plug-in, network service mesh, etc.) and other aspects.
```
Comment (dabernie) [inline whitepaper/chapter01.md]: "CNFs must be able to  run  on Kubernetes and benefit directly and strongly from cloud native principles."
I would remove the reference to k8s there since it is referred to in the previous sentence. Also, even if K8S is a flagship for CNCF, nothing prevents that in 2-3 years it might be replaced by a "shinier platform", cloud-native principles should still prevail.

File: whitepaper/chapter01.md [hunk 19]
```diff
@@ -0,0 +1,140 @@
...
+
+As cloud native involves decomposing tightly coupled processes into loosely coupled microservice, rearchitecting applications for CNFs often will entail decomposing functional aspects into services running in discrete containers or as separate microservices. For an example of how cloud native telecommunications principles might work in practice, consider a UTM (universal threat management) firewall. These are generally large physical or virtual appliances that encompasses multiple functions including: DDoS protection, IP filtering, VPN provisioning and termination, stateful packet inspection, and anti-virus / spam filtering. They are complicated to troubleshoot and a clear single point of failure for multiple critical functions. UTMs are also, by design, decidedly imperative - making them brittle and poor handlers of edge cases. UTMs cannot be easily managed via CI/CD and standard DevOps practices.
```
Comment (dabernie) [inline whitepaper/chapter01.md]: Ah ! My UTM use case ;-)

File: whitepaper/chapter01.md (hunk 1 above)
Comment (ASalkever) [reply whitepaper/chapter01.md]: My suggestion is we revert back to the original introduction with Bell Canada. It is a nice anecdote and frames the story. As follows:

If you use voicemail on a wireless network powered by Bell Canada, your message queue is officially cloud native. In 2019, Bell Canada ported its voicemail network functions over to a container-based architecture running on top of a Kubernetes orchestration layer alongside its production instance of the ONAP architecture. Voicemail is one of a number of functions that Bell Canada has or will be migrating to Cloud Native Network Functions (CNFs). “The main driver behind cloud native for Bell Canada is service availability, namely, how can we leverage cloud native principles to make sure our services are more agile, more flexible, and more resilient?,” explains Daniel Bernier, a Senior Technical Architect at Bell Canada and a member of Cloud Native Computing Foundation’s (CNCF) Telecom User Group (TUG). 

Telecommunications companies provide some of the most critical services in the world. From emergency 911 services to air traffic control to buy-sell signals that move billions of dollars worth of bonds, telecommunications networks impact our safety, social stability, and economic prosperity. s. Because the stakes are so high, telecommunications infrastructure has been designed, maintained, and upgraded to minimize risks and maximize reliability and uptime. 

Operators like Bell Canada intend to move more of their functionality to cloud native architectures as the telecommunications community further evolves standards and practices around deployment of mission critical applications and services over CNFs. 

The goal of this whitepaper is to provide an introduction to cloud native principles as they might apply to telecom operators and enterprises running internal telecommunications-like infrastructure.  By helping operators and others in the industry to better understand what cloud native means for telecommunications, we can help build consensus around industry adoption of cloud native technologies and all enjoy the benefits of this transition.

File: whitepaper/chapter01.md [hunk 20]
```diff
@@ -0,0 +1,140 @@
...
+
+<a name="1.3.1"></a>
+### 1.3.1 What "loosely coupled" means in cloud native systems
+
+Cloud native systems have a clear separation between the microservices. The separation is based on a technology like containers which allows independent life cycle management of each microservice. Cloud native applications, which are built on these microservices, and systems generally benefit from segregating as many services as possible into different containers. Additional benefits of cloud native architectures include scalability (vertical and horizontal) per microservice. This simplifies deployment in a wide variety of infrastructure environments, ranging from endpoint device to edge station to large data centres. It also allows service providers to deploy resources in the most efficient manner possible while maintaining necessary service levels for regulated and unregulated service types.
```
[the remark this answers is not in the document]
Comment (taylor) [reply whitepaper/chapter01.md]: @jeffsaelens good point.  Do you have a suggestion?  Maybe a followup PR?

File: whitepaper/chapter01.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (taylor) [reply whitepaper/chapter01.md]: @jeffsaelens and @dabernie these are good ideas.  I suggest we add them to a new PR after merging this one

File: whitepaper/chapter01.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (taylor) [reply whitepaper/chapter01.md]: @jeffsaelens and @dabernie these are good points as well. 

Can these go into a new PR?

File: whitepaper/chapter01.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (taylor) [reply whitepaper/chapter01.md]: @dabernie @jeffsaelens  I think this is an excellent topic for the TUG calls, a github issue and PR.  I expect it to take more time and space that this comment area allows for.

[the remark this answers is not in the document]
Comment (taylor) [reply whitepaper/chapter01.md]: https://github.com/cncf/telecom-user-group/issues/33

File: whitepaper/chapter01.md (hunk 14 above)
Comment (taylor) [reply whitepaper/chapter01.md]: @michaelspedersen, let's update these a new PR unless you have suggestions for the explanation in another section.  I hesitate to remove it yet until we have the replacement.

File: whitepaper/chapter01.md (hunk 7 above)
Comment (taylor) [reply whitepaper/chapter01.md]: Added a ticket to followup on this https://github.com/cncf/telecom-user-group/issues/34

**Comment by taylor** [comment]
Merging as the first version.  

We can continue to make improves with new PRs and opening issues for discussing topics that are not ready for PRs.

File: whitepaper/chapter01.md [hunk 21]
```diff
@@ -0,0 +1,140 @@
...
+
+For Kubernetes specifically, the atomic unit of management is a Pod. A Pod is a set of application containers which are 
+co-located and co-scheduled on the same machine (Node in Kubernetes terms) which run with a shared context and the same Linux networking namespace. For telco (and other) cloud native applications, the Pod is the basic building block of deployment, scaling operations, and upgrades.
+
+Cloud native applications consist of sets of cloud native microservices. The configuration of these microservices is not changed after deployment (note, this does allow for changes in application data such as "add a new user"). New features for a cloud native application and the underlying Pods are rolled out as new artifacts and new configuration. Cloud native systems are configured declaratively. This means that the system configuration describes “what” a loosely coupled system should look like, not ”how” the system should be created. The “how” of the application is determined by the tooling (e.g. the orchestrator, operators, and CRDs).
+
+<a name="1.4"></a>
+## 1.4 Cloud Native Network Functions
```
Comment (ASawwaf) [inline whitepaper/chapter01.md]: 1.4 Cloud Native Network Functions 
it will make more sense if we transfer this subchapter to next chapter 1.5.x
as Network Fn into more Telco , and its contain clauses related to Telco ( EPC )

File: whitepaper/chapter01.md [hunk 22]
```diff
@@ -0,0 +1,140 @@
...
+
+<a name="1.5.1"></a>
+### 1.5.1 A Brief History of Virtualisation for Telcos
+
+Until the 1990’s, most functionality and applications for telecommunications resided primarily in physical hardware (aka ‘boxes’). While software powered these systems, the software functioned more like firmware and was not easy to abstract and run on multiple platforms in virtual environments.
+
+This fostered a “top-down” approach to telecommunications systems that were imperative rather than declarative. This approach was exceptionally resource intensive and expensive; redundancy literally meant acquiring multiple physical instances of required systems. Not surprisingly, this guiding architectural decision encouraged the tight bundling of numerous services and capabilities into individual physical systems. This resulted not only in inflated infrastructure and provisioning costs but also inefficient hyper-redundancy in system engineering for overlapping capabilities.
+
+<a name="1.5.2"></a>
+### 1.5.2 Software Defined Networking And The Emergence of VNFs
```
Comment (ASawwaf) [inline whitepaper/chapter01.md]: @tomkivlin , @taylor in my POV it will be better if we add  a brief on ETSI NFV, not for virtualization itself covering the both above points (1.5.1&1.5.2) 

I will assign it to myself and provide it

Comment (ASawwaf) [reply whitepaper/chapter01.md]: @tomkivlin , I can't assign it to myself, plz support here

Comment (tomkivlin) [reply whitepaper/chapter01.md]: You will need to create a new issue, and%or create a branch on a fork of this repo. This PR has been merged so you won't be able to do it on here.

Comment (taylor) [reply whitepaper/chapter01.md]: @ASawwaf it seems like what you are proposing should go into a new chapter or section that talks about ETSI NFV from a historical point of view or alternative approach for implementing telephony platforms and technologies. 

Whatever the case, as @tomkivlin said, this PR is merged.  Please create a github issue for further discussion and/or PR.
